components/brave_news/rust/lib.rs

```diff
@@ -26,10 +27,55 @@ mod ffi {
     }
 
     extern "Rust" {
+        fn strip_html(subject: &str) -> String;
         fn parse_feed_bytes(source: &[u8], output: &mut FeedData) -> bool;
     }
 }
 
+// Note: This function isn't likely to be perfect, but it does guarantee that
+// there will be no HTML tags in the output (as it strips both '<' and '>' from
+// from the output).
+fn strip_html(subject: &str) -> String {
```

Comment:
cc @thypon to get your take on this approach for html sanitization rather than using `voca_rs`

Reply:
Confirmed with @thypon on Slack :smile: